README.rst

```diff
@@ -62,6 +62,15 @@ To parse a STIX JSON string into a Python STIX object, use ``parse()``:
 
 For more in-depth documentation, please see `https://stix2.readthedocs.io/ <https://stix2.readthedocs.io/>`__.
 
+STIX 2.X Technical Specification Support
+----------------------------------------
+
+The `stix2` Python library is built to support multiple versions of the STIX
```

Comment:
May want to put an explicit statement of "This version of python-stix2 supports STIX 2.0 by default", and then change this statement as new versions of python-stix2 supporting newer versions of STIX" are released.